images/ubuntu/templates/ubuntu-20.04.arm64.pkr.hcl

```diff
@@ -452,6 +452,12 @@ build {
     scripts          = ["${path.root}/../scripts/ubicloud/setup-runner-user.sh"]
   }
 
+  provisioner "shell" {
+    environment_vars = ["HELPER_SCRIPTS=${var.helper_script_folder}"]
+    execute_command  = "sudo sh -c '{{ .Vars }} {{ .Path }}'"
+    scripts          = ["${path.root}/../scripts/ubicloud/install-cache-proxy.sh"]
+  }
```

Comment:
We've already run `scripts/ubicloud/setup-runner-user.sh`. Could you merge the provisioners in a similar way to the others?

```
  provisioner "shell" {
    environment_vars = ["HELPER_SCRIPTS=${var.helper_script_folder}"]
    execute_command  = "sudo sh -c '{{ .Vars }} {{ .Path }}'"
    scripts          = [
        "${path.root}/../scripts/ubicloud/setup-runner-user.sh",
        "${path.root}/../scripts/ubicloud/install-cache-proxy.sh"
    ]
  }

```